templates/fine-tune-stable-diffusion/README.md

```diff
@@ -184,10 +177,14 @@ from utils import (
 
 
 # Set environment variables across the entire cluster.
+ENV_VARS = {"HF_HOME": "/mnt/local_storage/huggingface"}
```

Comment:
Should this use `SUBJECT_IMAGES_PATH`?

Reply:
Nope, this is for the model download cache.